Install error

I ran the build instructions as specified in the readme. However, cmake flips out when getting to `Performing configure step for 'project_libautoconf'`.

Here is the full error. 
```
configure: error: cannot find sources (lib/autoconf/autoconf.m4) in /vagrant/amazon-kinesis-video-streams-producer-sdk-cpp/open-source/libautoconf/build/src/project_libautoconf or ..
CMakeFiles/project_libautoconf.dir/build.make:109: recipe for target 'build/src/project_libautoconf-stamp/project_libautoconf-configure' failed
make[2]: *** [build/src/project_libautoconf-stamp/project_libautoconf-configure] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/project_libautoconf.dir/all' failed
Makefile:83: recipe for target 'all' failed
make[1]: *** [CMakeFiles/project_libautoconf.dir/all] Error 2
make: *** [all] Error 2
CMake Error at CMake/Utilities.cmake:68 (message):
  CMake step for libautoconf failed: 2
Call Stack (most recent call first):
  CMakeLists.txt:81 (build_dependency)

```

Any help would be appreciated.

In the configure logs we get

`checking for GNU M4 that supports accurate traces... configure: error: no acceptable m4 could be found in $PATH` 

Looking into this now!

--- a/README.md
+++ b/README.md
@@ -34,7 +34,7 @@ To download run the following command:
 `git clone --recursive https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-cpp.git`
 
 Note: If you miss running git clone with --recursive, run `git submodule update --init` in the amazon-kinesis-video-streams-producer-sdk-cpp/open-source directory
-You will also need to install `pkg-config` and `CMake` and a build enviroment. If you are building the GStreamer plugin you will also need GStreamer and GStreamer (Development Libraries).
+You will also need to install `pkg-config`, `CMake`, `m4` and a build enviroment. If you are building the GStreamer plugin you will also need GStreamer and GStreamer (Development Libraries).
 
 Refer to the [FAQ](#FAQ) for platform specific instructions.
 

--- a/docs/linux.md
+++ b/docs/linux.md
@@ -1,3 +1,8 @@
+### Installing libraries needed to build
+
+* Ubuntu/Debian - `apt-get install cmake m4 git build-essential`
+* RHEL/CentOS - `yum install cmake git m4 && yum groupinstall 'Development Tools'`
+
 ### How to run sample applications for sending media to KVS using [GStreamer](https://gstreamer.freedesktop.org/):
 
 ##### Setting credentials in environment variables

